ci-operator/templates/openshift/openshift-azure/build-base-image.yaml

```diff
@@ -63,6 +65,8 @@ objects:
           cpu: 3
           memory: 4Gi
       volumeMounts:
+      - name: artifacts
```

Comment:
If you don't gather any artifacts from here, remove

Reply:
we gather artifacts ...not in the traditional sense but this allows us to at least keep container logs

Reply:
ack